README.md

```diff
@@ -11,10 +11,16 @@ Recipebox utilizes localStorage to store your recipe collection. You can add fur
 
 ## Table of Contents
 
+- [Pre-Requisites](#pre-requisites)
 - [Run the app](#run-the-app)
 - [Test the app](#test-the-app)
 - [Build the app](#build-the-app)
 
+## Pre-Requisites
+
+This project requires [Node.js](https://nodejs.org) 4.2.0 or later to be installed.<br>
```

Comment:
Isn't the requried node version for react cli higher? Please verify this.

You also should've made a separate commit were you updated the package.json. I think you will need to update the yarn.lock too, because I think TravisCI priotizes the yarn.lock over package-lock when both are present. Maybe I should remove the package-lock at some point.

Reply:
Great points, I will update.